package.json

```diff
@@ -10,7 +10,7 @@
     "espree.js"
   ],
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=4.0.0"
```

Comment:
Since we're doing a breaking change, should we change this to `>=6.0.0` to be the same as ESLint v5?